ortho_seq_code/gui/worker.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import traceback
+
+from PyQt5.QtCore import QRunnable, pyqtSignal, QObject, pyqtSlot
+
+
+class WorkerSignals(QObject):
+    '''
+    Defines the signals available from a running worker thread.
+
+    Supported signals are:
+
+    finished
+        No data
+
+    error
+        `tuple` (exctype, value, traceback.format_exc() )
+
+    result
+        `object` data returned from processing, anything
+
+    progress
```

Comment:
should we do a progress bar in the future? Probably not necessary right now...

Reply:
we can add in future